llm/ggml.go

```diff
@@ -532,7 +532,7 @@ func (llm GGML) GraphSize(context, batch uint64, kvCacheType string) (kv, partia
 // SupportsKVCacheType checks if the requested cache type is supported
 func (ggml GGML) SupportsKVCacheType(cacheType string) bool {
 	validKVCacheTypes := []string{"f16", "q8_0", "q4_0"}
-	return slices.Contains(validKVCacheTypes, cacheType)
+	return slices.Contains(validKVCacheTypes, strings.ToLower(cacheType))
```

Comment:
As an alternative, do we want to do this when we read the env var?

Reply:
Sure thing, incoming...

Reply:
Updated :)